packages/odyssey-react-mui/src/OdysseyProvider.tsx

```diff
@@ -25,14 +25,17 @@ import {
   OdysseyTranslationProvider,
   OdysseyTranslationProviderProps,
 } from "./OdysseyTranslationProvider";
+import { OktaSupportedLanguages } from "./OdysseyTranslationProvider.types";
 
-export type OdysseyProviderProps = OdysseyCacheProviderProps &
+export type OdysseyProviderProps<
+  SupportedLanguages extends string = OktaSupportedLanguages
+> = OdysseyCacheProviderProps &
   OdysseyThemeProviderProps &
-  OdysseyTranslationProviderProps & {
+  OdysseyTranslationProviderProps<SupportedLanguages> & {
     children: ReactNode;
   };
 
-const OdysseyProvider = ({
+const OdysseyProvider = <SupportedLanguages extends string>({
```

Comment:
no, this does not force anyone to pass a generic EVERY time because we have a default value further down